### PR TITLE
LPD-38491 Get the right groupId

### PR DESCRIPTION
**Pull request by robertoDiaz**
# What is this trying to solve?

[Fix DepotWCSiteIntegration#CanSelectDepotContent](https://liferay.atlassian.net/browse/LPD-38260)

Getting the right groupId when configuring a layoutSetPrototype

# How am I fixing it?

Getting the right groupId when configuring a web content display portlet in a layoutSetPrototype page 

# How can you verify that it works?

DepotWCSiteIntegration#CanSelectDepotContent should be fixed

### CODE REVIEW

**Comment by robertoDiaz** [comment]
ci:test:content-management

**Comment by robertoDiaz** [comment]
Captura de pantalla 2024-10-07 a las 20 04 23

Tests were fixed

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/model/DepotAssetRendererFactoryWrapper.java [hunk 1]
```diff
@@ -41,13 +45,19 @@ public DepotAssetRendererFactoryWrapper(
 		DepotApplicationController depotApplicationController,
 		DepotEntryLocalService depotEntryLocalService,
 		GroupLocalService groupLocalService,
-		SiteConnectedGroupGroupProvider siteConnectedGroupGroupProvider) {
+		SiteConnectedGroupGroupProvider siteConnectedGroupGroupProvider,
+		LayoutPrototypeLocalService layoutPrototypeLocalService,
+		LayoutPageTemplateEntryLocalService
+			layoutPageTemplateEntryLocalService) {
```
Comment (adolfopa) [inline modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/model/DepotAssetRendererFactoryWrapper.java:51]: These need to be sorted alphabetically (LayoutPage* < LayoutPrototype*).

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/model/DepotAssetRendererFactoryWrapper.java (hunk 1 above)
Comment (robertoDiaz) [reply modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/model/DepotAssetRendererFactoryWrapper.java]: Done

**Comment by adolfopa** [comment]
ci:forward

**Comment by robertoDiaz** [comment]
ci:forward

**Pull request closed without merging**